From 1b2661c5f4b6799377b45abe65d34806052da19a Mon Sep 17 00:00:00 2001
From: Erfan Abdi <erfangplus@gmail.com>
Date: Mon, 26 Oct 2020 07:10:41 +0330
Subject: [PATCH] hwcomposer: Reduce frame delay

---
 backend/hwcomposer/output.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backend/hwcomposer/output.c b/backend/hwcomposer/output.c
index 8c789cb7..6e9d15f6 100644
--- a/backend/hwcomposer/output.c
+++ b/backend/hwcomposer/output.c
@@ -65,7 +65,7 @@ static bool output_set_custom_mode(struct wlr_output *wlr_output, int32_t width,
 		return false;
 	}
 
-	output->frame_delay = 1000000 / refresh;
+	output->frame_delay = 500000 / refresh;
 
 	wlr_output_update_custom_mode(&output->wlr_output, width, height, refresh);
 	return true;
